test/components/rankers/test_transformers_similarity.py

```diff
@@ -378,6 +379,7 @@ def test_run(self, query, docs_before_texts, expected_first_text, scores):
         assert docs_after[2].score == pytest.approx(sorted_scores[2], abs=1e-6)
 
     @pytest.mark.integration
+    @pytest.mark.slow
```

Comment:
Same as mentioned below, no need to have multiple parametrized tests here, could just be one.

Reply:
done in https://github.com/deepset-ai/haystack/pull/9296/commits/a31d1574eeede2ea202b44b66e65f09af454e4f9